homeassistant/components/hassio/http.py

```diff
@@ -213,3 +214,10 @@ def _get_timeout(path: str) -> ClientTimeout:
     if NO_TIMEOUT.match(path):
         return ClientTimeout(connect=10, total=None)
     return ClientTimeout(connect=10, total=300)
+
+
+def should_compress(content_type: str) -> bool:
+    """Return if we should compress a response."""
+    if content_type.startswith("image/"):
+        return "svg" in content_type
```

Comment:
Aren't we now saying, if the image is a svg, compress it? If so, why are we compressing SVG (as they are usually quite efficient)

Reply:
svgs are text so they compress well. all other image formats are expected to be binary

Reply:
Aah, I was thinking about image compression so it confused me